.github/workflows/approval-workflow.yml

```diff
@@ -0,0 +1,16 @@
+name: Team approval workflow
+
+on: pull_request_review
+  
+jobs:
+  labelWhenApproved:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Label approved pull request
+      uses: abinoda/label-when-approved-action@1.0.5
+      env:
+        APPROVALS: "1"
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        ADD_LABEL: "approved"
```

Comment:
Can you change the label to "Code Review Approved" ?

Reply:
Part of Tutorial, can we do this after this tutorial update?

Reply:
Can I move forward?